src/components/datepicker/datePicker.js

```diff
@@ -212,8 +212,11 @@
 
     self.$scope.$on('md-calendar-change', function(event, date) {
       self.ngModelCtrl.$setViewValue(date);
+      self.date = date;
       self.inputElement.value = self.dateLocale.formatDate(date);
       self.closeCalendarPane();
+      self.resizeInputElement();
+      self.inputContainer.classList.remove(INVALID_CLASS);
```

Comment:
Hm no, it doesn't look like calling $render actually triggers the input handlers